smart-contracts/test/test_txSignatureAggregation.js

```diff
@@ -195,16 +219,267 @@ describe("Gas Cost Tests", function () {
             claimData,
             signatures
         );
-
       const receipt = await tx.wait();
-      console.log("~~~~~~~~~~~~\nTotal: ", Number(receipt.gasUsed));
+      const sum = Number(receipt.gasUsed);
+      
+      if(gasProfiling.use) {
+        gasProfiling.current.newBb = sum;
+        logGasDiff('DoublePeg :: New BridgeToken:', gasProfiling.newBb, sum);
+      } else {
+        console.log("~~~~~~~~~~~~\nTotal: ", Number(receipt.gasUsed));
+      }
 
       const newlyCreatedTokenAddress = await state.cosmosBridge.sourceAddressToDestinationAddress(state.token1.address);
       expect(newlyCreatedTokenAddress).to.be.equal(expectedAddress);
     });
+
+    it("should allow us to check the cost of submitting a batch prophecy claim lock", async function () {
+      // Add tokens 2 and 3 into white list
+      await batchAddTokensToEthWhitelist(state, [state.token2.address, state.token3.address])
```

Comment:
After we call this batch add, let's do an explicit check where we call the contract and ensure they are whitelisted.